src/common/SurgeSynthesizerIO.cpp

```diff
@@ -378,7 +378,12 @@ void SurgeSynthesizer::savePatch()
    fs::path filename = savepath;
    filename /= string_to_path(legalname + ".fxp");
 
-   if (fs::exists(filename))
+   bool checkExists = true;
+#if LINUX
+    // Overwrite prompt hangs UI in Bitwig 3.3
```

Comment:
Does “bitwig” match case on Linux? Just checking

Thanks!!

Reply:
Yep! I tried both "Bitwig" and "bitwig" and all lowercase turned out to be the one.

Reply:
Great thanks. Will merge when ci is done